### ISSUE TITLE
[eBUG] SkillDrops files not loaded

### ISSUE THREAD

**Issue by pierpaolocira** [akasolace/HO#204]
Just open "Preferences" from my IDE in MacOS and got the following exceptions:
```
 [Error]   SkillDrops: Failed to read skill drop file: keeper. prediction/skilldrops/keeper (No such file or directory)
 [Error]   SkillDrops: Failed to read skill drop file: defending. prediction/skilldrops/defending (No such file or directory)
 [Error]   SkillDrops: Failed to read skill drop file: playmaking. prediction/skilldrops/playmaking (No such file or directory)
 [Error]   SkillDrops: Failed to read skill drop file: winger. prediction/skilldrops/winger (No such file or directory)
 [Error]   SkillDrops: Failed to read skill drop file: scoring. prediction/skilldrops/scoring (No such file or directory)
 [Error]   SkillDrops: Failed to read skill drop file: passing. prediction/skilldrops/passing (No such file or directory)
 [Error]   SkillDrops: Failed to read skill drop file: setpieces. prediction/skilldrops/setpieces (No such file or directory)
```

The issue should be around line 242 of `core.training.SkillDrops`

**Comment by pierpaolocira** [comment]
To be done after  #205

**Comment by pierpaolocira** [comment]
This is already fixed on my branch, because it relates the fileLoader refactoring that I'm working on... I'll take care to mark any related issue in the pull request.

**Comment by akasolace** [comment]
closed by 92efca1
